WebHostLib/static/assets/playerTracker.js

```diff
@@ -1,33 +1,35 @@
 window.addEventListener('load', () => {
   // Reload tracker
   const update = () => {
-    // update items section
-    const item_columns = document.getElementsByClassName('item-column');
-    for (let i = 0; i < item_columns.length; i++) {
-      $( item_columns[i].id ).load(' ' + item_columns[i].id);
-    }
+    const url = document.getElementById('tracker-wrapper').getAttribute('data-tracker');
 
-    // update locations section
-    const location_columns = document.getElementsByClassName('location-column');
-    for (let i = 0; i < location_columns.length; i++) {
-      $( location_columns[i].id ).load(' ' + location_columns[i].id);
-    }
-
-    // update icons section
-    const icon_columns = document.getElementsByClassName('icon-column');
-    for (let i = 0; i < icon_columns.length; i++) {
-      $( icon_columns[i].id ).load(' ' + icon_columns[i].id);
-    }
-
-    // update regions section
-    const regions = document.getElementsByClassName('regions-column');
-    for (let i = 0; i < regions.length; i++) {
-      let region_name = regions[i].id;
-      $( region_name ).load(' ' + region_name);
-    }
+    fetch('/tracker/' + url, {
+        method: "GET"
+    }).then(function (response) {
+        return response.text()
+    }).then(function (html) {
+        document.body.innerHTML = html;
```

Comment:
Altering HTML directly is bad practice. You should instead create an endpoint on the WebHost that responds with a JSON file which you then use to build or update whatever DOM elements you need. In this case, it's probably better to build the DOM initially through Jinja, then create an endpoint which provides data used to update the DOM.